server/utils/viewUtils.ts

```diff
@@ -54,4 +56,16 @@ export default class ViewUtils {
       }),
     }
   }
+
+  static govukFormattedDateFromStringOrNull(date: string | null): string {
```

Comment:
Formatting a date is presenter logic - I'm not sure what this is doing in `ViewUtils`. If anywhere I'd say this should go in `ReferralDataPresenterUtils`.

Reply:
done